plugins/modules/aci_aaa_key_ring.py

```diff
@@ -52,6 +52,7 @@
   key:
     description:
     - The private key for the certificate.
+    - Providing this option will always result in a change because it is a secure property that cannot be retrieved from APIC.
```

Comment:
Did we validate these cases in the tests? Should we write tests for this or adding note in description is good enough?

Reply:
Thanks for suggesting, I'll add it in. :)

Reply:
Done, found 9 modules that required extra tests.